.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: xenial
 os: linux
 
 before_install:
-  - npm install eslint
+  - npm install eslint@6.6.0
```

Comment:
I just got stuck on this yesterday for a PR in Wikilabels.  I came to the same solution.